data/data/ovirt/variables-ovirt.tf

```diff
@@ -46,27 +46,26 @@ variable "ovirt_network_name" {
   description = "The name of ovirt's logical network for the selected ovirt cluster."
 }
 
-variable "ovirt_master_mem" {
-  type    = string
-  default = "8192"
+variable "ovirt_master_memory" {
```

Comment:
please include descriptions for these global variables.

Reply:
done